contracts/rewards/src/error.rs

```diff
@@ -1,8 +1,12 @@
 use axelar_wasm_std_derive::IntoContractError;
+use cosmwasm_std::{OverflowError, StdError};
 use thiserror::Error;
 
 #[derive(Error, Debug, PartialEq, IntoContractError)]
 pub enum ContractError {
+    #[error(transparent)]
+    Std(#[from] StdError),
```

Comment:
if at all possible, I'd like to avoid a catchall Std error below the root level. This makes it too easy to default wrapping everything in this which contradicts the purpose of error-stack to add context to the errror